numpy/random/bit_generator.pyi

```diff
@@ -0,0 +1,61 @@
+from threading import Lock
+from typing import Literal, Optional
+
+from numpy import DtypeLike, ndarray
+from numpy.typing import ArrayLike, _ShapeLike
+
+class SeedlessSeedSequence:
+    def generate_state(
+        self, n_words: int, dtype: DtypeLike = ...
+    ) -> ndarray: ...
+    def spawn(self, n_children: int) -> List[SeedlessSeedSequence]: ...
+
+class SeedSequence:
```

Comment:
A number of attributes and methods are missing:
* `entropy`
* `spawn_key`
* `pool_size`
* `n_children_spawned`
* `pool`
* `mix_entropy()`
* `get_assembled_entropy()`

Reply:
Some of these are `cdef` which means not available in Python.  Should these be typed as well (e.g., `mix_entropy()`).

Reply:
Ah, I wasn't aware of that. If they're not available in Python then they don't have to be typed, no.